collector/agentPoller.go

```diff
@@ -341,28 +327,25 @@ func (a *Agent) getSystemMetrics(agentState *AgentState, stats chan<- statsd.Sta
 }
 
 // fetches container state from the agent (via /state) to populate AgentState
-func (a *Agent) getAgentState(stats chan<- statsd.StatsEvent) (*AgentState, error) {
-	rootJSON, err := a.getJSONFromAgent("/state", agentTestStateFileFlag, stats)
+func (a *Agent) getAgentState() (*AgentState, error) {
+	rootJSON, err := a.getJSONFromAgent("/state", agentTestStateFileFlag)
 	if err != nil {
 		return nil, err
 	}
 
 	json, err := rootJSON.Object()
 	if err != nil {
-		stats <- statsd.MakeEvent(statsd.AgentQueryBadData)
 		return nil, err
 	}
 
 	// state["id"] (agent_id)
 	agentID, err := json.GetString("id")
 	if err != nil {
-		stats <- statsd.MakeEvent(statsd.AgentQueryBadData)
 		return nil, err
 	}
 
 	frameworks, err := json.GetObjectArray("frameworks")
```

Comment:
Same here, and in the other instances of setting these values (per above comment).